README.md

```diff
@@ -203,21 +202,25 @@ There are some rules for prefixing to observe:
 
 `Sanitizer`:
 
-* Determines prefix as any text (including whitespace) that comes before `REPOBEE-SANITIZER-START`
-* Only determines prefix on a block-to-block basis, meaning that the prefix
+* Removes prefixes on `REPLACE-WITH` blocks when sanitizing
+  * Only the first occurrence of a prefix on each line is removed, allowing code comments to be preserved when sanitizing
+* Determines the prefix as any text before `REPOBEE-SANITIZER-START`
+  * This excludes any whitespace before the `START` marker
+    * Pro-Tip: There can be any amount of whitespace on any line using a prefix, see example 4. This allows easy alignment of code
+  * When the prefix is removed after running `Sanitizer`, all whitespace is preserved, only the prefix is removed
+* Determines prefix on a block-to-block basis, meaning that the prefix
```

Comment:
This is very hard to read to me. I don't think using bullet point lists to present information this complex is appropriate. The bullet point list here prior to this PR was okayishly readable, but this is just way too much for my eyes. Have a look in the rendered document, can you see what I mean?

Although not really related to this PR (but this is the first time I see this text), I noticed that the text leading up to the bullet point list is also pretty strange, with two colon-terminated half-sentences like so:

```
There are some rules for prefixing to observe:

Sanitizer:
```

And not all of the bullet points can even be read as "Sanitizer <does X>", for example `Sanitizer: Code between replace and end markers ..."` doesn't make sense.

I think this whole thing would be much better described as fluent text describing how a prefix is defined, and then how it is used. Special cases could then potentially be documented in a bullet point list. For example, it could be structured something like so:

> Prefixes are valid for a single block, and are defined on the same line as the `REPOBEE-SANITIZER-START` marker of that block. All text starting from the first non-whitespace up until (but not including) the start marker counts as a prefix, and is valid for the rest of that block. Blabla blablabla blabla.

Reply:
The indented points are related to its parent so it makes sense for me when I wrote it like that, but thinking about how others would perceive it I think you're right. Also i can agree that it's just too much right now, so ill rewrite it to text